From ab40da7bfd1c505aa081a4da0b559ada74a2d36b Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Mon, 1 Sep 2014 09:25:17 -0400
Subject: [md] raid5: fix "distingush" typo

Message-id: <1409563584-9000-16-git-send-email-Jes.Sorensen@redhat.com>
Patchwork-id: 89953
O-Subject: [RHEL7.1 PATCH 15/82] treewide: fix "distingush" typo
Bugzilla: 1085530
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

From: Michael Opdenacker <michael.opdenacker@free-electrons.com>

Signed-off-by: Michael Opdenacker <michael.opdenacker@free-electrons.com>
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
(partial backport of aa5e5dc2a8878ecf1a94819d889939023fd576c9)
Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/raid5.h b/drivers/md/raid5.h
index 2113ffa..b42e6b4 100644
--- a/drivers/md/raid5.h
+++ b/drivers/md/raid5.h
@@ -49,7 +49,7 @@
  * can't distinguish between a clean block that has been generated
  * from parity calculations, and a clean block that has been
  * successfully written to the spare ( or to parity when resyncing).
- * To distingush these states we have a stripe bit STRIPE_INSYNC that
+ * To distinguish these states we have a stripe bit STRIPE_INSYNC that
  * is set whenever a write is scheduled to the spare, or to the parity
  * disc if there is no spare.  A sync request clears this bit, and
  * when we find it set with no buffers locked, we know the sync is
-- 
1.7.1